### DISCUSSION TITLE
How to monitor github actions waiting and running time

### DISCUSSION THREAD

**Discussion by david-gang** [community/community#25903] [Actions]
<p>Jenkins can show us the build time trend. Is there a similar functionality in github actions?<br>
Additionally is there a way to fetch the jobs waiting for a runner? I want to get this info and see if i need to add runners. It would be even greater if i would get such data programmatically in real time so that i could make autoscaling based on this.</p>

**Comment by NicoNBW** [comment, marked as answer]
<p>Thanks for your feedback. Currently no such a built-in rule to achieve your requirement. However the requirement makes sense. I recommend that you can directly report the feature requests <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=actions" rel="nofollow noopener">here</a>. That will allow you to directly interact with the appropriate engineering team, and make it more convenient for the engineering team to collect and categorize your suggestions.</p>
